tests/pass-dep/libc/libc-random.rs

```diff
@@ -61,3 +63,12 @@ fn test_getrandom() {
         );
     }
 }
+
+#[cfg(any(target_os = "solaris", target_os = "illumos", target_os = "solaris"))]
+fn test_arc4random_buf() {
+    let mut buf = [0u8; 5];
+    unsafe {
+        libc::arc4random_buf(buf.as_mut_ptr(), buf.len());
+        assert!(buf[0] != 0u8);
```

Comment:
This can fail, if we happen to get all-zero... 1/256 is not *so* unlikely, that means it'll fail on rustc CI around once a month. So let's do:
```suggestion
        assert!(buf != [0u8; 5]);
```

Reply:
true